### PR TITLE
Updated tgocassis2isis to import the output of tgocassisrdrgen

### PR DESCRIPTION
**Pull request by krlberry**

### CODE REVIEW

File: isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp [hunk 1]
```diff
@@ -12,12 +12,14 @@
 #include "UserInterface.h"
 #include "XmlToPvlTranslationManager.h"
 #include "iTime.h"
+#include "IException.h"
```
Comment (jcwbacker) [inline isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp]: please alphabetically order #includes

File: isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp [hunk 2]
```diff
@@ -41,7 +43,14 @@ void IsisMain() {
     }
     
     Cube *outputCube = importer.SetOutputCube("TO");
-    translateLabels(xmlFileName, outputCube);
+
+    QString transRawFile("/translations/tgoCassisInstrument.trn");
```
Comment (jcwbacker) [inline isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp]: intialize with equal sign

https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html#declaring

File: isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp [hunk 3]
```diff
@@ -41,7 +43,14 @@ void IsisMain() {
...
+    QString transExportFile("/translations/tgoCassisExportedInstrument.trn");
+    try {
+      translateLabels(xmlFileName, outputCube, transRawFile); 
+    } catch (IException &e) {
```
Comment (jcwbacker) [inline isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp]: catch on new line
https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html#controlStructures

File: isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp [hunk 4]
```diff
@@ -81,12 +90,33 @@ void translateCoreInfo(FileName &inputLabel, ProcessImport &importer) {
   PvlGroup &dataDir = Preference::Preferences().findGroup("DataDirectory");
   QString missionDir = (QString) dataDir["Tgo"];
 
+  // How to tell that this is an ISIS3 exported product (generated by tgocassisrdrgen?)
```
Comment (jcwbacker) [inline isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp]: i have added something to ProcessExportPds4 to address this issue (for my hyb1 ticket). You can remove this comment.

File: isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp [hunk 5]
```diff
@@ -81,12 +90,33 @@ void translateCoreInfo(FileName &inputLabel, ProcessImport &importer) {
...
+
   // Get the translation manager ready
-  FileName transFile(missionDir + "/translations/tgoCassis.trn");
-  XmlToPvlTranslationManager labelXlater(inputLabel, transFile.expanded());
+  FileName transFile; 
+  try {
+    transFile = FileName(missionDir + "/translations/tgoCassis.trn"); 
+    XmlToPvlTranslationManager labelXlater(inputLabel, transFile.expanded());
+    translateCoreInfo(labelXlater, importer);
+  } catch (IException &e) {
```
Comment (jcwbacker) [inline isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp]: newline

**Review by jcwbacker — CHANGES_REQUESTED** [review]
Added a few coding style suggestions.
I will do more in-depth review tomorrow.
Also, did you add any new tests and check code coverage yet for these changes? (I didn't see any Makefile changes)

**Comment by krlberry** [comment]
@jcwbacker Thanks! I fixed the coding-style comments you mentioned. In terms of new tests and test coverage, Makayla and I decided to just add another card to Trello for adding a new test to test these change, since it seemed like perhaps there wasn't enough cassis work for everyone to do.

**Comment by krlberry** [comment]
@jcwbacker Actually, never mind! We're going to add the test now.

File: isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp [hunk 6]
```diff
@@ -82,11 +114,31 @@ void translateCoreInfo(FileName &inputLabel, ProcessImport &importer) {
...
+ * Translate core info from labels and set ProcessImport object with 
+ * these values.
+ *
+ * @param labelXlater Reference to the XmlToPvlTranslationManager objcet to use for the translation.
+ * @param importer Reference to the ProcessImport object to which core info will
+ *                 be set.
+ */
+void translateCoreInfo(XmlToPvlTranslationManager labelXlater, ProcessImport &importer) {
   // Set up the ProcessImport
+  QString str;
```
Comment (celbal) [inline isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp:141]: is str just a temporary string?

Comment (krlberry) [reply isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp:141]: Yes

**Review by celbal — APPROVED** [review]

**Pull request merged**